IDARE/app/src/main/java/com/opensource/app/idare/viewmodel/MainActivityViewModel.java

```diff
@@ -31,18 +35,36 @@ public boolean onNavigationItemSelected(MenuItem menuItem, FragmentTransaction f
             mPreviousMenuItem.setChecked(false);
         }
         mPreviousMenuItem = menuItem;
-        AppTourFragment appTourFragment = AppTourFragment.newInstance();
+        ActiveProfileFragment activeProfileFragment = ActiveProfileFragment.newInstance();
         Fragment fragment = null;
         switch (menuItem.getItemId()) {
+            case R.id.active_profile:
+                fragment = activeProfileFragment;
+                break;
+            case R.id.core_list:
+                CoreListFragment coreListFragment = CoreListFragment.newInstance();
+                fragment = coreListFragment;
+                break;
+            case R.id.invite:
+                InviteToIDareFragment inviteToIDareFragment = InviteToIDareFragment.newInstance();
+                fragment = inviteToIDareFragment;
+                break;
             case R.id.app_tour:
+                AppTourFragment appTourFragment = AppTourFragment.newInstance();
                 fragment = appTourFragment;
                 break;
-            case R.id.active_profile:
-                ActiveProfileFragment activeProfileFragment = ActiveProfileFragment.newInstance();
-                fragment = activeProfileFragment;
+            case R.id.donate:
+                DonateFragment donateFragment = DonateFragment.newInstance();
+                fragment = donateFragment;
+                break;
+            case R.id.settings:
+                SettingsFragment settingsFragment = SettingsFragment.newInstance();
+                fragment = settingsFragment;
+                break;
+            case R.id.log_out:
                 break;
             default:
-                fragment = appTourFragment;
+                fragment = activeProfileFragment;
                 break;
         }
         dataListener.replaceFragment(fragment);
```

Comment:
this should be in a if condition (only when fragment is not null)

Reply:
OK fine